### DISCUSSION TITLE
is there a gui that works with this?

### DISCUSSION THREAD

**Discussion by ralyodio** [mudler/LocalAI#978] [Q&A]
I'm looking for both the server side api type this repo provides, as well as a graphical web/desktop interface. Does such a thing exist?

**Comment by rafacampoamor** [comment]
A newbie here interested on this topic as well. Is there any interface (no matter if web, or desktop app to compile) to work with it?

**Comment by lunamidori5** [comment]
@ralyodio and @rafacampoamor 
Every thing that works with ``OpenAI`` will work with ``LocalAI`` Check out our site for more info - https://localai.io/integrations/
